compiler/rustc_const_eval/src/interpret/step.rs

```diff
@@ -270,18 +270,23 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 M::box_alloc(self, &dest)?;
             }
 
-            NullaryOp(mir::NullOp::SizeOf, ty) => {
+            NullaryOp(null_op, ty) => {
```

Comment:
Please match `mir::NullOp::SizeOf | mir::NullOp::AlignOf` here.  so that the match remains properly exhaustive and will catch additional `NullOp` additions in the future.

Reply:
How about change the _ arm below matching null_op to a NullOp::Box arm instead? This way we still get exhaustive match and is IMO cleaner.

Reply:
I don't have a strong opinion either way.